File: app/src/main/java/com/arny/lubereckiy/models/Locations.java

package com.arny.lubereckiy.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import io.realm.RealmObject;

public class Locations  extends RealmObject {

    @SerializedName("parent")
    @Expose
    private Parent parent;
    @SerializedName("child")
    @Expose
    private Child child;

    public Parent getParent() {
        return parent;
    }

    public void setParent(Parent parent) {
        this.parent = parent;
    }

    public Child getChild() {
        return child;
    }

    public void setChild(Child child) {
        this.child = child;
    }

}

File: app/src/main/java/com/arny/pik/data/models/Locations.kt
package com.arny.pik.data.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Locations {
    @SerializedName("parent")
    @Expose
    var parent: Parent? = null
    @SerializedName("child")
    @Expose
    var child: Child? = null

}
